src/main/java/org/moltimate/moltimatebackend/service/CacheService.java

```diff
@@ -0,0 +1,57 @@
+package org.moltimate.moltimatebackend.service;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.RemovalCause;
+import lombok.extern.slf4j.Slf4j;
+import org.moltimate.moltimatebackend.dto.Alignment.QueryAlignmentResponse;
+import org.moltimate.moltimatebackend.repository.FailedAlignmentRespository;
+import org.moltimate.moltimatebackend.repository.QueryAlignmentResponseRepository;
+import org.moltimate.moltimatebackend.repository.QueryResponsetDataRepository;
+import org.moltimate.moltimatebackend.repository.SuccessfulAlignmentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+
+@Service
+@Slf4j
+public class CacheService {
+    @Autowired
+    private QueryAlignmentResponseRepository queryAlignmentResponseRepository;
+
+    @Autowired
+    private QueryResponsetDataRepository queryResponsetDataRepository;
```

Comment:
Also rename the variable once you fix the typo

Reply:
Done